--- mpb-d7d4930ebe84c5ca9abe750021e106e204ab79ae/examples/check_old.ctl	2015-07-30 16:18:24.200297686 -0700
+++ mpb-d7d4930ebe84c5ca9abe750021e106e204ab79ae/examples/check.ctl	2015-07-30 16:27:00.702983146 -0700
@@ -56,7 +56,7 @@
 
 ; checks whether list X and list Y are almost equal
 (define (check-almost-equal X Y)
-  (if (fold-left and true (map almost-equal? X Y))
+  (if (fold-left (lambda (new prev) (and new prev)) true (map almost-equal? X Y))
       (print "check-almost-equal: PASSED\n")
       (error "check-almost-equal: FAILED\n" X Y)))
 
